com.vrcfury.vrcfury/Runtime/VF/Component/VRCFuryHapticSocket.cs

```diff
@@ -103,7 +104,7 @@ public class DepthAction {
         }
 
         public override int GetLatestVersion() {
-            return 7;
+            return 8;

```

Comment:
This should also not change. Beware that this will break any sockets that you've upgraded to this new version 8.

Reply:
yep, pretty much.